lib/xen_api.ml

```diff
@@ -113,18 +113,20 @@ module Make(IO:IO) = struct
         Response.read_body_chunk reader
         >>= function
         | Cohttp.Transfer.Chunk x ->
+          (* Printf.eprintf "Chunk: %s\n%!" x; *)
```

Comment:
I assume we could remove these now that it works?

Reply:
No, I'd rather keep them for the next time :P